apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list-public-page/community-events-list-public-page.component.scss

```diff
@@ -1,3 +1,6 @@
-a {
-  margin: 0.5rem auto;
+.main-div {
+  @apply com-flex com-flex-wrap;
+  a {
+    margin: 0.5rem auto;
+  }
 }
```

Comment:
try to use better class names
and use tailwind classes

Reply:
Done